config/kinto.ini

```diff
@@ -75,6 +74,9 @@ multiauth.policies = basicauth
 # kinto.statsd_url = udp://localhost:8125
 # kinto.statsd_prefix = kinto-prod
 
+# kinto.http_scheme = http
+# kinto.http_host = 0.0.0.0:8888
```

Comment:
This should never look like that, I should be ``kinto.http_host = localhost:8888``